apps/schools/domains/common/components/styles/abstracts/_colors.scss

```diff
@@ -75,3 +89,11 @@ $color-header-table: #F2F3F7;
 $color-search-highlight-text: #ffc95f;
 $color-background: #434343;
 $color-foreground: rgba(67, 67, 67, 0.80);
+
+//base rgba with opacity
+$color-black-01: rgba(0, 0, 0, 0.10);
+$color-black-015: rgba(0, 0, 0, 0.15);
+$color-black-02: rgba(0, 0, 0, 0.20);
+$color-white-01: rgba(255, 255, 255, 0.1);
+$color-white-015: rgba(255, 255, 255, 0.15);
+$color-white-02: rgba(255, 255, 255, 0.2);
```

Comment:
it is better to group similar colors together
for example black colors in one group and white in another